src/reducers/query-reducer.ts

```diff
@@ -62,6 +63,20 @@ export function reducer(
             set(draft, resourceId, resource);
           }
           return;
+
+        case RESOURCE_DEL_SUCCESS:
+          {
+            action = action as ResourceSuccessAction;
+            resource.loading = false;
+            resource.initialized = true;
+            resource.error = null;
+            resource.data = (<DataType[]>resource.data).filter(
+              i => i.id !== action.payload.serviceParameters.id
+            );
+            resource.meta = action.payload.meta || null;
+            set(draft, resourceId, resource);
```

Comment:
This is operating over a single query. I don't think that is going to work. I need to go over all the queries for this resourceType.

Reply:
yes I see now. My bad I only looked at the first one. You are correct let me change that. I understand what you mean.

Reply:
Something on these lines? 
`Object.values(get(draft, `${resourceType}`)).forEach(res => {
              res.data = (<DataType[]>res.data).filter(
                i => i.id !== action.payload.serviceParameters.id
              );
            });`

Reply:
https://gist.github.com/oizie/ab800d1a73ed5d640261e392a8c7ea5d

Reply:
I would add a test for this as well to cover this scenario since we don't have any tests for the query-reducer.

Reply:
cool cool

Reply:
Please take a look now. Just added a test as well.